### PR TITLE
Add create

### PR DESCRIPTION
**Pull request by kjcsb1**

### CODE REVIEW

File: applications/crossbar/doc/storage.md [hunk 1]
```diff
@@ -223,7 +223,77 @@ Key | Description | Type | Default | Required | Support Level
...
 `connection` |   | `string()` |   | `false` |  
 
+#### Create
 
+First, we need a 32-character UUID which will be the ID of the S3 configuration that we'll use later in the storage plan setup. Creating a UUID is easy to do with the following cURL command:
+
+```shell
+curl https://www.uuidgenerator.net/api/version4/1 | sed 's/-//g'
+e3e26e06a4b1465599e6dc9e1516fd8b
+```
```
Comment (lazedo) [inline applications/crossbar/doc/storage.md]: can we remove the reference to `https://www.uuidgenerator.net` and add instead  `echo $(tr -dc a-f0-9 < /dev/urandom | dd bs=32 count=1 2> /dev/null)`

File: applications/crossbar/doc/storage.md [hunk 2]
```diff
@@ -223,7 +223,77 @@ Key | Description | Type | Default | Required | Support Level
...
+                }
+            }
+        }
+        }}' \
+  http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/storage
+```
+
+```json
+{
+    "auth_token": "{AUTH_TOKEN}"
```
Comment (icehess) [inline applications/crossbar/doc/storage.md]: missing comma `,` at the end of the line.

**Comment by lazedo** [comment]
@kjcsb1 ping

**Comment by jamesaimonetti** [comment]
@kjcsb1 I updated the text a bit and merged this. Thanks!

**Pull request closed without merging**